### DISCUSSION TITLE
GitHub Achievements

### DISCUSSION THREAD

**Discussion by ChewyToast** [community/community#28459] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everone!

Just asking for the eazyest achivements to get on github!

tx

**Comment by xRodzMC** [comment, marked as answer]
The pair, extraordinaire

Like mine, just make pull request, and merge with other person.

Adding commit merge message:
```
Co-authored-by: github-username <example@github.com>
```

> **Warning**
> Co-authored Email should valid.

The example is on https://github.com/ReinfyTeam/ProfanityFilter/pull/32

Take look my profile, for proff i claim easily this badge. 😎

![Screenshot_2022-08-05-18-13-23-13_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/100104970/183056278-224cabc5-b146-449c-b93d-14fc4fe1370d.jpg)

> **Note**
> If you found this helpful or answer your question just make me as answer.

**Comment by radar-blip** [comment]
Here is a full list of achievement badges and how to get them:
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by ChewyToast** [comment]
thx to everyone!

**Reply by xRodzMC** [reply]
no problem <3 we are here to help
